### PR TITLE
new hunter gear cosmetics!

### PR DESCRIPTION
**Pull request by Nomoresolvalou**
# About the pull request
adds in 1 new cloak halfarmor sprite, and 5 new masks

# Explain why it's good for the game

amazing kino sprites from Thwomper soulify the game experience

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

masks in the bronze style
image

new mask in bone style plus new armor in crimson style
image

</details>

# Changelog
:cl: Nomoresolvalou, Thwomper
add: Added 1 new hunter armor and 5 new hunter masks
imageadd: Added sprites for the new cosmetics, made by Thwomper
/:cl:

### CODE REVIEW

**Comment by Zenith00000** [comment]
Please include a screenshot of the new stuff. Although personally I know amazing stuff, Thwomper is a legend.

**Comment by Nomoresolvalou** [comment]
> Please include a screenshot of the new stuff
done

**Review by Blundir — APPROVED** [review]

File: maps/templates/lazy_templates/clf_ert_station.dmm
Comment (Nanu308) [inline maps/templates/lazy_templates/clf_ert_station.dmm:1]: lgtm

File: maps/predship/huntership.dmm
Comment (Nanu308) [inline maps/predship/huntership.dmm:1]: lgtm

File: maps/map_files/New_Varadero/New_Varadero.dmm
Comment (Nanu308) [inline maps/map_files/New_Varadero/New_Varadero.dmm:1]: lgtm

File: maps/map_files/LV624/maintemple/2.flooded.dmm
Comment (Nanu308) [inline maps/map_files/LV624/maintemple/2.flooded.dmm:1]: lgtm

File: maps/map_files/LV624/maintemple/1.intact.dmm
Comment (Nanu308) [inline maps/map_files/LV624/maintemple/1.intact.dmm:1]: lgtm

File: maps/map_files/LV624/LV624.dmm
Comment (Nanu308) [inline maps/map_files/LV624/LV624.dmm:1]: lgtm

File: maps/map_files/DesertDam/sprinkles/10.damtemple_intact.dmm
Comment (Nanu308) [inline maps/map_files/DesertDam/sprinkles/10.damtemple_intact.dmm:1]: lgtm

File: maps/map_files/DesertDam/Desert_Dam.dmm
Comment (Nanu308) [inline maps/map_files/DesertDam/Desert_Dam.dmm:1]: lgtm

**Review by Nanu308 — APPROVED** [review]

**Pull request merged**